### DISCUSSION TITLE
Is it possible to select all options in a checkbox prompt ?

### DISCUSSION THREAD

**Discussion by ony-boom** [c4spar/cliffy#648] [Q&A] [answered]
Hello 👋.
Thanks for cliffy✨.

I was wondering **how can we select all options in a `checkbox prompt`**. 
Is this already implemented or do we have to do it manually?

Thank you!

**Comment by c4spar** [comment, marked as answer]
Hi, no this is currently not implemented but we could add a shortcut to select/deselect all options.

**Reply by ony-boom** [reply]
OK,  thanks ✨
